api-tests/server/settings_test.go

```diff
@@ -158,8 +128,8 @@ func TestSettings(t *testing.T) {
 
 				res, err := serverClient.Default.ServerService.ChangeSettings(&server.ChangeSettingsParams{
 					Body: server.ChangeSettingsBody{
-						EnableStt:        true,
-						DisableTelemetry: true,
+						EnableStt:       true,
```

Comment:
Can we rename this to `EnableAdvisors` now? It no longer reflects the reality IMO and only a few would even remember what `stt` refers to :)

Reply:
We have another ticket for that: https://perconadev.atlassian.net/browse/PMM-12677

Michael already working on renaming